src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__day-1 calendar__day-1--start-day"></div>
+      <div class="calendar__day-2"></div>
+      <div class="calendar__day-3"></div>
+      <div class="calendar__day-4"></div>
```

Comment:
Use css for numbers. All blocks should have only `calendar__day` class
<img width="572" alt="image" src="https://user-images.githubusercontent.com/99403198/214555436-1112eb2d-7897-4931-a91a-479359e3fc76.png">
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```